### PR TITLE
NEWS: Update

### PR DESCRIPTION
**Pull request by shamisp**
Fixing news before RC2 tag.

### CODE REVIEW

**Comment by shamisp** [comment]
@yosefe please take a look.

**Comment by shamisp** [comment]
I will tag RC2 and send out announcement in an hour

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8723/ for details (Mellanox internal link).

File: NEWS [hunk 1]
```diff
@@ -30,7 +21,12 @@ Features:
 - GPU-to-Infiniband HCA affinity support based on locality/distance (PCIe)
 
 Bugfixes:
-- Multiple bugfixes (full list on github)
+- Fix overflow in RC/DC flush operations
+- Update description in SPEC file and README
+- Fix RoCE source port for dc_mlx5 flow control
+- Improve ucx_info help message
+- Fix segfault in UCP, due to int truncation in count_one_bits()
+- Multiple other bugfixes (full list on github)
```
Comment (manjugv) [inline NEWS]: @shamisp This seems relevant here, but you are repeating this again on line 36 (you might want to remove it).

Comment (shamisp) [reply NEWS]: @manjugv thanks, copy-n-past error. The one on line 36 has to be removed

**Pull request merged**